src/Mod/Fem/Gui/Command.cpp

```diff
@@ -1129,10 +1132,191 @@ bool CmdFemCreateNodesSet::isActive()
 
 //================================================================================================
 //================================================================================================
+
+/* start of Erase Element bit */
+DEF_STD_CMD_A(CmdFemDefineElementsSet);
+
+void DefineElementsCallback(void *ud, SoEventCallback *n)
+{
+    Fem::FemAnalysis* Analysis;
+
+//    Base::Console().Warning("DefineElemensCall\n");
+
+    if (getConstraintPrerequisits(&Analysis))
+        return;
+
+    // show the wait cursor because this could take quite some time
+    Gui::WaitCursor wc;
+
+    // When this callback function is invoked we must in either case leave the edit mode
+//    Gui::View3DInventorViewer* view = reinterpret_cast<Gui::View3DInventorViewer*>(n->getUserData());
+    Gui::View3DInventorViewer* view = static_cast<Gui::View3DInventorViewer*>(n->getUserData());
+    view->setEditing(false);
+    view->removeEventCallback(SoMouseButtonEvent::getClassTypeId(), DefineElementsCallback, ud);
+    n->setHandled();
+
+    Gui::SelectionRole role;
+    std::vector<SbVec2f> clPoly = view->getGLPolygon(&role);
+    if (clPoly.size() < 3)
```

Comment:
Please comments so that other developers who might n e.g. 2 years need to  fix an issue, can understand what is done?
For you at the moment it is clear why the if is for < 3, but others in future would need this info